react/components/InfoCard/index.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent } from 'react'
+import { bool, string, oneOf } from 'prop-types'
+import classNames from 'classnames'
+import { withRuntimeContext } from 'vtex.render-runtime'
+
+import CallToAction from './CallToAction'
+import { textPositionTypes, textAlignmentTypes, callToActionModeTypes, textPostionValues, textAlignmentValues } from './SchemaTypes'
+
+import styles from './infoCard.css'
+
+const justifyTokens = {
+  [textPostionValues.LEFT]: 'justify-start',
+  [textPostionValues.CENTER]: 'justify-center',
+  [textPostionValues.RIGHT]: 'justify-end'
+}
+
+const alignTokens = {
+  [textAlignmentValues.LEFT]: 'tl',
+  [textAlignmentValues.CENTER]: 'tc',
+  [textAlignmentValues.RIGHT]: 'tr'
+}
+
+const itemsTokens = {
+  [textAlignmentValues.LEFT]: 'items-start',
+  [textAlignmentValues.CENTER]: 'items-center',
+  [textAlignmentValues.RIGHT]: 'items-end'
+}
+
+const flexOrderTokens = {
+  [textPostionValues.LEFT]: 'flex-row',
+  [textPostionValues.RIGHT]: 'flex-row-reverse'
+}
+
+const defaultValues = {
+  textPosition: textPositionTypes.TEXT_POSITION_LEFT.value,
+  textAlignment: textAlignmentTypes.TEXT_ALIGNMENT_LEFT.value,
+}
+
+const getEnumValues = enumObject => Object.values(enumObject).map(({ value }) => value)
+const getEnumNames = enumObject => Object.values(enumObject).map(({ name }) => name)
+
+const safelyGetToken = (tokenMap, valueWanted, propName) => tokenMap[valueWanted] || defaultValues[propName]
+
+class InfoCard extends PureComponent {
+  static propTypes = {
```

Comment:
Personally, I think this looks better outside the component.

Reply:
I disagree hahah (just my 2c though)